src/schedule.js

```diff
@@ -0,0 +1,42 @@
+import { RRule } from 'rrule';
+
+class ScheduleUtils {
+  static occursNow(rruleString, durationSeconds) {
+    const rrule = RRule.fromString(rruleString.replace("\\n", "\n"));
+    const duration = durationSeconds * 1000;
+
+    const now = new Date();
+
+    // For evaluation with the RRule library we pretend that "now" is in UTC instead of the local timezone.
```

Comment:
```suggestion
    // From the RRULE docs:_ "Returned "UTC" dates are always meant to be 
    // interpreted as dates in your local timezone. This may mean you have to 
    // do additional conversion to get the "correct" local time with offset 
    // applied."
    //
    // We do the opposite to ensure that datetime comparisons works as expected. 
    // For evaluation with the RRule library we pretend that "now" is in UTC instead of the local timezone.
```